scripts/utils.py

```diff
@@ -102,3 +104,32 @@ def get_remote(exec_path, remote_addr) -> str:
 
         remote = [remote for remote in remotes if remote_addr in remote][0]
         return remote.split()[0]
+
+
+def find_file() -> str:
+    """cross platform find"""
+    return ""
+
+
+def get_o_spreadsheet_hash(path) -> str:
+    # TORORAR make it crossplatform
+    lines = (
+        subprocess.check_output(["tail", "-10", path])
```

Comment:
probably worth adding a comment to explain this.
Also, what is "path" ? is it `o_spreadsheet.js` path ?
could be named `o_spredsheet_path` then